src/rosetta-helpers.ts

```diff
@@ -282,6 +283,15 @@ export function isDecimalsSupported(operations: RosettaOperation[]): boolean {
   return true;
 }
 
+export function getCurrencyData(): RosettaCurrency {
```

Comment:
Maybe this can be renamed to `getStxCurrencyMetadata()`, or something a little more specific. Reading the fn name elsewhere in the code I wasn't entirely sure what it returns.